src/ember-app.js

```diff
@@ -12,6 +13,11 @@ const debug = require('debug')('fastboot:ember-app');
 const FastBootInfo = require('./fastboot-info');
 const Result = require('./result');
 
+const LATEST_SCHEMA_VERSION = '1';
+// this allows fastboot lib to have unpublished features from ember-cli-fastboot but allows
+// people to run integration tests in ember-cli-fastboot
+const NEXT_SCHEMA_VERSION = '2';
```

Comment:
Perhaps the schema version should be represented as an integer, so `NEXT_SCHEMA_VERSION` can be `LATEST_SCHEMA_VERSION++` and updating the version doesn't require editing two values?